requirements/_requirements.txt

```diff
@@ -18,3 +18,4 @@ requests-toolbelt>=1.0.0
 wheel>=0.38.1
 setuptools>=65.5.1
 pytest-asyncio<=0.21.1
+orjson
```

Comment:
maybe a better placement would be requirements.http.txt?

Reply:
Agreed! And can we pin orjson?